Add faq that insert function can only be connected to one destination (#5203)
segmentio/segment-docs: src/connections/functions/insert-functions.md

No, destination insert functions are currently available as cloud-mode destinations only. Segment is in the early phases of exploration and discovery for supporting customer "web plugins" for custom device-mode destinations and other use cases, but this is unsupported today.

##### Can I connect an insert function to multiple destinations?

> No, a Destination Insert Function currently can be connected to one destination only.

Reviewer: ```suggestion
No, an insert function can only be connected to one destination.
```
